plugin/src/functionalTest/groovy/com/here/gradle/plugins/jobdsl/JobDslPluginTest.groovy

```diff
@@ -58,7 +58,7 @@ class JobDslPluginTest extends Specification {
 
         then:
         project.repositories.find { it.url.toString() == 'https://repo.jenkins-ci.org/releases/' }
-        project.repositories.find { it.url.toString() == 'https://repo1.maven.org/maven2/' }
+        project.repositories.find { it.url.toString() == 'https://repo.maven.apache.org/maven2/' }
```

Comment:
Could you please explain in the commit message why this change is also necessary when upgrading Gradle?

Reply:
Done.